content/release-10-7-0/application-10-7-0-bundle/10-7-0-24.md

```diff
@@ -250,7 +250,7 @@ MTM-31421</td>
 <tr>
 <td>
 Cockpit</td>
-<td > The "Map" widget now does not display more devices than configured as default pageSize for the map ( e.g. 40).
+<td > The "Map" widget now does not display more devices than configured as default pageSize for the map ( for example, 40).
```

Comment:
is the space character after the opening bracket necessary?

Reply:
No, good catch! Fixed